bmwqemu.pm

```diff
@@ -284,6 +284,7 @@ sub log_call {
             push @result, join("=", $key, pp($value));
         }
         $params = join(", ", @result);
+        $params =~ s/^/  /gm;
```

Comment:
This actually adds two spaces at the beginning of the string, which is probably not what we want.
I thought `s/(?<!\A)^/  /gm` could do the trick by explicitly excluding `\A` as the beginning of the string.

Reply:
or: `s/\n(?=\S)/\n  /g;`

with the advantage that empty lines (or lines containing only spaces) won't be indented.

Reply:
Thanks, interesting stuff